domains/transfer/Transfer.API/API/Program.cs

```diff
@@ -85,9 +92,66 @@
 
 builder.Services.AddLogging();
 
+
 builder.Services.AddTransfer(builder.Configuration);
 builder.Services.AddCvr();
 builder.Services.AddConnection();
+builder.Services.AddClaimServices();
+
+builder.Services.AddControllers(options => options.Filters.Add<AuditDotNetFilter>())
+    .AddJsonOptions(options =>
+        options.JsonSerializerOptions.Converters.Add(new JsonStringEnumConverter()));
+Configuration.Setup()
+    .UseEntityFramework(ef => ef
+        .AuditTypeExplicitMapper(config => config
+            .Map<TransferAgreement, TransferAgreementHistoryEntry>((evt, eventEntry, historyEntity) =>
+            {
+                var actorId = evt.CustomFields.ContainsKey("ActorId")
+                    ? evt.CustomFields["ActorId"].ToString()
+                    : null;
+                var actorName = evt.CustomFields.ContainsKey("ActorName")
+                    ? evt.CustomFields["ActorName"].ToString()
+                    : null;
+
+                historyEntity.Id = Guid.NewGuid();
+                historyEntity.CreatedAt = DateTimeOffset.UtcNow;
+                historyEntity.AuditAction = eventEntry.Action;
+                historyEntity.ActorId = actorId ?? string.Empty;
+                historyEntity.ActorName = actorName ?? string.Empty;
+
+                switch (eventEntry.Action)
+                {
+                    case "Insert":
+                        historyEntity.TransferAgreementId = (Guid)eventEntry.ColumnValues["Id"];
+                        break;
+                    case "Update":
+                        {
+                            historyEntity.TransferAgreementId = (Guid)eventEntry.PrimaryKey.Values.First();
+                            break;
+                        }
+                }
+
+                return true;
+            })
```

Comment:
Could this be moved into the startup-file for TransferAgreement? The same goes for the claim part below

Reply:
You cannot have two configurations of audit.net. Dno how to split them, got any ideas?

Reply:
You can use an extension method for that